### ISSUE TITLE
Syncing of contacts and calendar to native android is no longer possible.

### ISSUE THREAD

**Issue by michaelkarrer81** [Odoo-mobile/messaging#21]
With the Version 2 of the messaging app it is no longer possible to sync partners or calendar entries to the native android contacts or calendar.

There is no longer an option available in the android account for odoo to sync contacts or calndar. Using the Sync server Partners checkbox does nothing for me?

Is this functionality dropped? - If so it would sadly render itself useless for me.

**Comment by michaelkarrer81** [comment]
@dpr-odoo  Any hint on this? I really do not want to be rude but it would help a lot to know if this features are dropped! Thank you.

**Comment by dpr-odoo** [comment]
Yes, Contacts and Calendar features are dropped from Messaging App v2.

We have planned to make separate app for **Calendar**. And Contacts (partners) sync are available in CRM App when released.

Reagards,

**Comment by michaelkarrer81** [comment]
Thank you very much for your answer. Even if it is not exactly what i was hoping for. 
If you need Beta Testers - count me in (we have a load of different android devices)!
